ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py

```diff
@@ -269,3 +269,29 @@ def test_paddle_logical_xor(
         x=x[0],
         y=x[1],
     )
+    
+   
+@handle_frontend_test(
+    fn_tree="paddle.logical_not",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"), num_arrays=1, shared_dtype=True
```

Comment:
The `num_arrays` is set to `1` by default so you can remove that. The `shared_dtype` argument is used when we are generating more than `1` arrays so you can remove that as well.